src/map/ai/helpers/action_queue.cpp

```diff
@@ -97,8 +97,14 @@ bool CAIActionQueue::isEmpty()
     return actionQueue.empty() && timerQueue.empty();
 }
 
-void CAIActionQueue::clearQueue()
+void CAIActionQueue::clearActionQueue()
 {
     while (!actionQueue.empty())
         actionQueue.pop();
 }
+
+void CAIActionQueue::clearTimerQueue()
+{
+    while (!timerQueue.empty())
+        timerQueue.pop();
```

Comment:
For style purposes:  Braces on newlines:
```
while (!timerQueue.empty())
{
    timerQueue.pop();
}
```
(Yes, I know the actionQueue loop isn't styled this way --  this also helps us delineate old code from new code quickly)

Reply:
Thanks!